crates/autopilot/src/lib.rs

```diff
@@ -497,9 +497,17 @@ pub async fn main(args: arguments::Arguments) {
     };
     let serve_metrics = shared::metrics::serve_metrics(Arc::new(liveness), args.metrics_address);
 
+    let auction_transaction_updater = crate::auction_transaction::AuctionTransactionUpdater {
+        web3,
+        db,
+        current_block: current_block_stream,
+    };
+    let auction_transaction = tokio::task::spawn(auction_transaction_updater.run_forever());
+
     tokio::select! {
-        result = serve_metrics => tracing::error!(?result, "serve_metrics exited"),
-        _ = db_metrics => unreachable!(),
-        _ = maintenance_task => unreachable!(),
+        result = serve_metrics => panic!("serve_metrics excited {:?}", result),
+        result = db_metrics => panic!("db_metrics excited {:?}", result),
+        result = maintenance_task => panic!("maintenance excited {:?}", result),
+        result = auction_transaction => panic!("auction_transaction excited {:?}", result),
```

Comment:
These fibers are very excited 😛.

```suggestion
        result = serve_metrics => panic!("serve_metrics exited {:?}", result),
        result = db_metrics => panic!("db_metrics exited {:?}", result),
        result = maintenance_task => panic!("maintenance exited {:?}", result),
        result = auction_transaction => panic!("auction_transaction exited {:?}", result),
```